dotcom-rendering/src/web/components/App.tsx

```diff
@@ -236,8 +240,9 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 				incrementWeeklyArticleCount(storage.local, CAPI.pageId);
 			}
 		};
-		incrementArticleCountsIfConsented().catch((e) =>
-			console.error(`incrementArticleCountsIfConsented - error: ${e}`),
+
+		setArticleCountIsReady(
+			incrementArticleCountsIfConsented().then(() => true),
```

Comment:
I don't love the logic here of chaining on `incrementArticleCountIfConsented` but I couldn't come up with a better option. The rendering of the banners/epics is blocked as long as `articleCountIsReady` is `undefined`, so we don't want to wait before calling `setArticleCountIsReady` e.g by doing 

```ts
useEffect(() => {
		const incrementArticleCountsIfConsented = async () => {
			const hasOptedOut = await hasOptedOutOfArticleCount();
			if (!hasOptedOut) {
				incrementDailyArticleCount();
				incrementWeeklyArticleCount(storage.local, CAPI.pageId);
			}
            setArticleCountIsReady(Promise.resolve(true));
		};
```

As the promise isn't getting us anything here. We can't do it in a separate hook e.g

```ts
useEffect(() => {
  setArticleCountIsReady(hasOptedOutOfArticleCount().then(() => true));
}, [])
```
As I think we have a race condition with actually incrementing the article counts. By chaining on `incrementArticleCountIfConsented` we can guarantee that we'll access the article counts after we update them